webapp/app.py

```diff
@@ -69,21 +67,93 @@ def results():
     return flask.render_template("careers/results.html", **context)
 
 
-@app.route("/careers/admin", methods=["GET", "POST"])
-@app.route("/careers/all", methods=["GET", "POST"])
-@app.route("/careers/commercial-ops", methods=["GET", "POST"])
-@app.route("/careers/design", methods=["GET", "POST"])
-@app.route("/careers/engineering", methods=["GET", "POST"])
-@app.route("/careers/finance", methods=["GET", "POST"])
-@app.route("/careers/hr", methods=["GET", "POST"])
-@app.route("/careers/legal", methods=["GET", "POST"])
-@app.route("/careers/marketing", methods=["GET", "POST"])
-@app.route("/careers/project-management", methods=["GET", "POST"])
-@app.route("/careers/sales", methods=["GET", "POST"])
-@app.route("/careers/tech-ops", methods=["GET", "POST"])
-def department_group():
-    department = flask.request.path.split("/")[2]
-    vacancies = greenhouse_api.get_vacancies(department)
+# Class that collects department-specific content
+class Department(object):
+    __careers_directory = "./templates/careers"
+
+    def __parse_feed_department(feed_department):
+        field = {
```

Comment:
I think `field_mapping` would be a better name here.

Reply:
Sorry the function should remain `__parse_feed_department` its the object `field` that should be renamed `field_mapping`